app/views/person/history/_history_members_form_sbv.html.haml

```diff
@@ -0,0 +1,34 @@
+.accordion
+  %a.accordion-toggle.collapsed.header{ href: "#new_role", data: { toggle: :collapse } }
+    = t('.new_role')
+
+
+  .accordion-body.collapse{ id: 'new_role' }
+    .accordion-inner{ style: 'height: 350px;'}
+      %fieldset
+        %legend
+        - @group_selection = Group::Verein.all
```

Comment:
das werden circa 2.000 Vereine. Und ich sehe auch nicht, wo die Daten verwendet werden. Wenn sie in dem Dropdown verwendet werden, dann würde ich lieber nur die Vereine laden, die sichtbar sind. Im Falle des Dachverbandes sind das wieder alle, aber bei allen anderen wären es deutlich weniger... Was meinst du?

Reply:
Ja die werden im Dropdown verwendet. Die Überlegung ist gut, ich glaube aber dass es nicht so einfach funktioniert. Die Vereine werden ja im Verlauf auf der Person geladen. Ich habe dort keinen richtigen scope auf eine Gruppe, sondern nur auf die Person selbst und muss deshalb aus meiner Ansicht auch immer vom Dachverband ausgehen. Sprich ich muss in jedem Fall die 2000 Vereine laden. 

Ich weiss, dass die Person theoretisch eine Primary-Group hat. Von der können wir aber schlecht ausgehen, da ansonsten keine Rolle im Verlauf oberhalb dieser Gruppe erstellt werden kann.

Reply:
Moment, ich glaube, wir hatten mit SBV vereinbart, dass es besser ist, das nicht über die echten Vereine zu machen. Es ist möglich, dass er in einem Verein war, der nie in der DB existiert hat. Die Idee war glaube ich, dass man da relativ frei eintragen kann, ohne dass man echte Rollen generiert. Ich kläre das grad mit Andre

Reply:
Aber auch sonst müssen dort sowieso alle Vereine als Möglichkeit bereitstehen. Ein Musiker kann ja in seinem Leben von Kanton zu Kanton umziehen.